pkg/cmd/server/server.go

```diff
@@ -640,6 +641,14 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 	}
 
 	deletionControllerRunInfo := func() controllerRunInfo {
+		var snapshotterClient *snapshotterClientSet.Clientset
+		var err error
+		if features.IsEnabled(features.EnableCSI) {
+			snapshotterClient, err = snapshotterClientSet.NewForConfig(s.kubeClientConfig)
```

Comment:
once #2323 merges, you probably want to reuse the CSI snapshot client that is being instantiated [here](https://github.com/vmware-tanzu/velero/pull/2323/files#diff-cd1a009fa409db8496ea35b53f9776d2R286)

Reply:
💯 I wanted to get this change tested out and did what was best to avoid being blocked on #2323